app/src/main/java/com/stonks/ui/currency/CurrencyFragmentUtils.kt

```diff
@@ -0,0 +1,216 @@
+package com.stonks.ui.currency
+
+import android.util.Log
+import android.widget.EditText
+import android.widget.TextView
+import com.github.mikephil.charting.charts.LineChart
+import com.stonks.api.currency.CurrencyApiDataUtils
+import com.stonks.ui.chart.StockLineChart
+import io.reactivex.disposables.Disposable
+import java.math.BigDecimal
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+const val DEFAULT_CURRENCY_INDEX = 17
+const val PRIMARY_RATES = "USD,EUR,GBP,JPY,CHF"
+
+class CurrencyFragmentUtils(disposable: Disposable?) {
+    private val TAG = CurrencyFragmentUtils::class.java.name
+    private var localDisposable = disposable
+
+    /**
+     * Умножает две строки [firstNumber]  и [secondNumber] и возвращает только первые 7 символов
+     */
```

Comment:
If you're documenting functions it's a good idea to specify what are their params and return values. Not really a deadly sin here, but just for future & to refactor later

Reply:
Accordint to [this](https://stackoverflow.com/questions/38015307/generating-kotlin-method-class-comments/51067319#51067319) SO answer it is better to comment with this style